packages/amplication-server/src/core/resource/resource.service.ts

```diff
@@ -609,17 +609,37 @@ export class ResourceService {
   }
 
   async gitRepository(resourceId: string): Promise<GitRepository | null> {
-    return await this.prisma.gitRepository.findUnique({
-      where: {
-        resourceId
-      },
-      include: {
-        gitOrganization: true
-      }
-    });
+    return (
+      await this.prisma.resource.findUnique({
+        where: { id: resourceId },
+        include: { gitRepository: { include: { gitOrganization: true } } }
+      })
+    ).gitRepository;
+  }
+
+  async gitOrganizationByResource(
+    args: FindOneArgs
+  ): Promise<GitOrganization | null> {
+    return (
+      await this.prisma.resource.findUnique({
+        ...args,
+        include: { gitRepository: { include: { gitOrganization: true } } }
+      })
+    ).gitRepository.gitOrganization;
   }
 
   async project(resourceId: string): Promise<Project> {
-    return this.prisma.project.findUnique({ where: { id: resourceId } });
+    return this.prisma.resource
+      .findUnique({ where: { id: resourceId } })
+      .project();
+  }
```

Comment:
we agreed that we should remove this function and the resolver should use the project service instead of the resource service